scormxblock/scormxblock.py

```diff
@@ -967,7 +970,7 @@ def update_scorm_status(self, data, version):
                           raw_possible=(info['maxi'] - info['mini']))
 
         if score:
-            if not self.has_submitted_answer() or self.allows_rescore():
```

Comment:
- When first score update come, the `self.scorm_status` will be assigned with value IN_PROCESS / info['status']